index.html

```diff
@@ -44,9 +44,49 @@
         </div>
       </div>
     </header>
+  
     
 
     <!-- Block banner -->
-    <main class="banner"></main>
+  <main class="banner">
+      <div class="main-container">
+      
+  
+      <div class="column-lg-9">
+        <h1 class="main__title">Fullstack-разработка<br> на JavaScript</h1>
+      
+        <div class="main__text">Пройдя курс Fullstack-разработки на JavaScript ты научишься<br>использовать профессиональные инструменты front-end и back-end<br>разработки React и Node.js, СУБД MySQL и MongoDB.</div>
+    
+      <div>
+        <ul class="main__list">
+          <li class="main__list-platform"><b>Площадка:</b><br>ННГУ</li>
+          <li class="main__list-time"><b>Продолжительность:</b><br>4 месяца</li>
+          <li class="main__list-date"><b>Дата:</b><br>29 февраля 2020</li>
+          <li class="main__list-price"><b>Стоимость:</b><br>26 000 ₽</li>
+        </ul>
+      </div>
+
+      
+        <button class="main__button" type="button">ЗАПИСАТЬСЯ</button>
```

Comment:
it's better to use this CSS property to make text uppercase https://doka.guide/css/text-transform/